managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -912,67 +904,6 @@ public long getTotalSize() {
         return TOTAL_SIZE_UPDATER.get(this);
     }
 
-    @Override
-    public void checkBackloggedCursors() {
```

Comment:
I would still keep this optimization where broker will not cache the entry at all if cursor is not caught up because there is no point to add entries for backlogged cursor because they are anyway not going to use it.

Reply:
Sure, I'll leave it back.